package.json

```diff
@@ -52,15 +41,23 @@
     "babel-preset-env": "^1.7.0",
     "eslint": "^5.16.0",
     "eslint-config-prettier": "^2.9.0",
-    "eslint-plugin-prettier": "^2.6.1",
-    "husky": "^1.0.0-rc.9",
+    "husky": ">=4",
     "jest": "^23.1.0",
-    "lint-staged": "^7.2.0",
+    "lint-staged": ">=10",
     "prettier": "^1.19.1",
     "rimraf": "^2.7.1",
     "rollup": "^0.62.0",
     "rollup-plugin-babel": "^3.0.5",
     "rollup-plugin-uglify": "^4.0.0",
     "vuepress": "^0.12.0"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.js": "eslint --cache --fix",
+    "*.{js,css,md}": "prettier --write"
   }
```

Comment:
Don't we need the `git add` step in this new configuration?

Reply:
This was changed in [`lint-staged` v10.0.0](https://github.com/okonet/lint-staged/releases/tag/v10.0.0).